DeploymentTemplates/aws/BasicSetup.yaml

```diff
@@ -30,6 +38,14 @@ Parameters:
     AllowedPattern: "^[a-zA-Z0-9]+[-a-zA-Z0-9-]+[-a-zA-Z0-9]+$"
     Description: "Role used by the Lambda."
 
+  sourceVpcId:
+    Type: "String"
+    Default: ""
```

Comment:
Would it be possible to ask for the VPC id, only if we select Private endpoints as apiGatewayType? This field shows up even when I pick Region endpoints as the apiGatewayType

sourceVpcId
Snowflake VPC that has access to private API Gateway. Used only when creating a private API Gateway

Reply:
i looked online, but it doesn't seem like cloudformation supports optional parameters :(

Reply:
gotcha. no worries. would have been nice, but should be fine.